core/web/apiv2/auth.py

```diff
@@ -130,7 +131,7 @@ def __init__(self, admin: bool):
     def __call__(self, user: User = Depends(get_current_user)) -> User:
         if not user.admin:
```

Comment:
If I correctly understand, it's only to validate admin access. Should we rename the class to something more relevant?